about.html

```diff
@@ -0,0 +1,191 @@
+          <div class="overlay-content">
+            <a href="./index.html">Home</a>
+            <a href="#about-us" onclick="closeNav()">About Us</a>
+            <a href="#past-activities" onclick="closeNav()">Past Activities</a>
+            <a href="#contact" onclick="closeNav()">Contact</a>
+          </div>
+        </div>
+        <div class="desktop-navbar">
+          <a href="./index.html" class="main-logo"
+            ><img
+              src="./Assets/images/Stem logo.png"
+              alt="main logo"
+              class="main-logo-img"
+          /></a>
+          <ul class="navbar-ul">
+            <li>
+              <a href="./index.html" class="desktop-nav-link">Main Page</a>
+            </li>
+            <li>
+              <a href="#about-us" class="desktop-nav-link">About Us</a>
+            </li>
+            <li>
+              <a href="#past-activities" class="desktop-nav-link"
+                >Past Activities</a
+              >
+            </li>
+            <li><a href="#contact" class="desktop-nav-link">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="container">
+        <img
+          src="./Assets/images/about-bg.png"
+          alt="Background image"
+          class="bg-header-image"
+        />
+        <div class="main-content">
+          <div class="main-content-headers flex-column-center">
+            <h2 class="special-color text-center">"Who are we?"</h2>
+            <h1 class="special-color text-center">
+                JAVASCRIPT JUNCTION: UNITING DEVELOPERS FOR A CODE-INFUSED SOCIETY
+            </h1>
+          </div>
+          <div class="header-content text-center margin-top-big">
+            <p>
+                JavaScript Society is a community of JavaScript enthusiasts, developers, and professionals who come together to learn, share knowledge, and collaborate on all things related to JavaScript.  We are a diverse group of individuals with a common passion for JavaScript and its applications in web development, mobile app development, server-side scripting, and more.
+            </p>
+            <p>
+            <b><i> Never Stop Innovating </i></b>
+            </p>
+          </div>
+          <div class="infos">
+            <p class="location light-black text-center">
+              <i
+                >Please Contact Us with Email for any further questions about
+                this activity!</i
+              ><br /><b>info@jsociety.org</b>
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about-us flex-column-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <img
+        src="./Assets/images/about us background.PNG"
+        alt="about us community photo"
+        class="family-photo"
+      />
+      <div class="about-us-content">
```

Comment:
- [ ] Please make sure that you use the layout of provided design including placement of the content. In order to follow the design guideline I would suggest you to make the about us section image size smaller and place it in the end of the section and not the beginning of it 

Cindy's 
---
<img width="919" alt="Screenshot 2023-07-07 at 12 14 15 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/ae994d85-a8c3-436f-837e-1b5769afbeca">

Your's
----
![FireShot Capture 063 - STEM Impact Congress - 2023 - 127 0 0 1](https://github.com/peter-mwovi/Capstone-project/assets/44798044/34c799e4-69f1-454e-a429-e7ed940b0e7b)